README.md

```diff
@@ -12,13 +12,14 @@
 
 ### Online sessions
 
-Due to COVID-19, we're going to conduct online sessions at least from 16th to 27th of March.
+Due to COVID-19, we're going to conduct online sessions throughout the semester.
```

Comment:
We dont know yet

Reply:
How about "Due to COVID-19, we're going to conduct online sessions until the next notice of the school."?

Reply:
Id rather delete phrases from "at least ...".

Reply:
I've fixed it.